css-fonts-3/font-style-matching-manual.html

```diff
@@ -0,0 +1,2712 @@
+
+}
+
+
+
+    #ultraexpanded_oblique_fivehundred {
+    font-family: CSSMatchingTest;
+    font-stretch: ultra-expanded;
+    font-style: oblique;
+    font-weight: 500;
+
+}
+
+
+
+    #ultraexpanded_oblique_sixhundred {
+    font-family: CSSMatchingTest;
+    font-stretch: ultra-expanded;
+    font-style: oblique;
+    font-weight: 600;
+
+}
+
+
+
+    #ultraexpanded_oblique_sevenhundred {
+    font-family: CSSMatchingTest;
+    font-stretch: ultra-expanded;
+    font-style: oblique;
+    font-weight: 700;
+
+}
+
+
+
+    #ultraexpanded_oblique_eighthundred {
+    font-family: CSSMatchingTest;
+    font-stretch: ultra-expanded;
+    font-style: oblique;
+    font-weight: 800;
+
+}
+
+
+
+    #ultraexpanded_oblique_ninehundred {
+    font-family: CSSMatchingTest;
+    font-stretch: ultra-expanded;
+    font-style: oblique;
+    font-weight: 900;
+
+}
+
+
+
+    #ultraexpanded_italic_onehundred {
+    font-family: CSSMatchingTest;
+    font-stretch: ultra-expanded;
+    font-style: italic;
+    font-weight: 100;
+
+}
+
+
+
+    #ultraexpanded_italic_twohundred {
+    font-family: CSSMatchingTest;
+    font-stretch: ultra-expanded;
+    font-style: italic;
+    font-weight: 200;
+
+}
+
+
+
+    #ultraexpanded_italic_threehundred {
+    font-family: CSSMatchingTest;
+    font-stretch: ultra-expanded;
+    font-style: italic;
+    font-weight: 300;
+
+}
+
+
+
+    #ultraexpanded_italic_fourhundred {
+    font-family: CSSMatchingTest;
+    font-stretch: ultra-expanded;
+    font-style: italic;
+    font-weight: 400;
+
+}
+
+
+
+    #ultraexpanded_italic_fivehundred {
+    font-family: CSSMatchingTest;
+    font-stretch: ultra-expanded;
+    font-style: italic;
+    font-weight: 500;
+
+}
+
+
+
+    #ultraexpanded_italic_sixhundred {
+    font-family: CSSMatchingTest;
+    font-stretch: ultra-expanded;
+    font-style: italic;
+    font-weight: 600;
+
+}
+
+
+
+    #ultraexpanded_italic_sevenhundred {
+    font-family: CSSMatchingTest;
+    font-stretch: ultra-expanded;
+    font-style: italic;
+    font-weight: 700;
+
+}
+
+
+
+    #ultraexpanded_italic_eighthundred {
+    font-family: CSSMatchingTest;
+    font-stretch: ultra-expanded;
+    font-style: italic;
+    font-weight: 800;
+
+}
+
+
+
+    #ultraexpanded_italic_ninehundred {
+    font-family: CSSMatchingTest;
+    font-stretch: ultra-expanded;
+    font-style: italic;
+    font-weight: 900;
+
+}
+
+
+
+
+</style>
+</head>
+<body>
+    <h1>CSS Font Style Matching Test</h1>
+    <p>This test tests for steps <a href="http://www.w3.org/TR/css3-fonts/#font-style-matching">4. a-c of the font style
+    matching algorithm</a>. Stretch, style and weight information specified on the left-hand side should match the
+    information on the right-hand side.</p>
+<p>This test works by declaring @font-face rules for each combination of stretch, style and weight and refering to a
+    different src url for each such rules. The font that the src url points to has a special property: it renders
```

Comment:
Agreed that it would be much easier to review the tool than the somewhat verbose tests it generated.